modules/gltf_blend/editor_scene_importer_blend.cpp

```diff
@@ -0,0 +1,228 @@
+	String apply_modifications_arg = int32_t(p_options["blend/nodes/modifiers"]) == BLEND_MODIFIERS_ALL ? "export_apply=True," : "export_apply=False,";
+	String common_args = vformat("filepath='%s',", parameters["sink"]) +
+			vformat("export_texture_dir='%s',", texture_dir_global) +
+			"export_format='GLTF_SEPARATE',"
+			"export_yup=True," +
+			visible_arg +
+			extra_properties_arg +
+			apply_modifications_arg +
+			deform_bones_args +
+			colors_arg +
+			uvs_arg +
+			skins_arg +
+			influence_arg +
+			animation_always_sample_arg +
+			animation_limit_playback_arg +
+			animation_groups_arg +
+			cameras_arg +
+			lights_last_arg;
+	String scene_to_blend_setting = "import bpy, os, sys;" + vformat("bpy.ops.wm.open_mainfile(filepath='%s');", parameters["source"]) +
+			"(bpy.ops.export_scene.gltf(export_keep_originals=True," + common_args + ")) "
+																					 "if bpy.app.version >= (3, 0, 0) else (bpy.ops.export_scene.gltf(" +
+			common_args + "));";
+	String script = scene_to_blend_setting;
+	Vector<String> args;
+	args.push_back("--background");
+	args.push_back("--python-expr");
+	args.push_back(script);
+	print_line(script);
+	Array standard_out;
+	int32_t ret = core_bind::OS::get_singleton()->execute(addon_path, args,
+			standard_out, true);
+	for (int32_t line_i = 0; line_i < standard_out.size(); line_i++) {
+		print_line(standard_out[line_i]);
+	}
+	if (ret != OK) {
+		*r_err = ERR_SCRIPT_FAILED;
+		print_error("Blender returned " + itos(ret));
+		return nullptr;
+	}
+	return import_scene_from_other_importer(sink,
+			p_flags, p_options, p_bake_fps);
+}
+
+Ref<Animation> EditorSceneFormatImporterBlend::import_animation(
+		const String &p_path, uint32_t p_flags, const Dictionary &p_options,
+		int p_bake_fps) {
+	return Ref<Animation>();
+}
+
+Variant EditorSceneFormatImporterBlend::get_option_visibility(
+		const String &p_path, const String &p_option,
+		const Map<StringName, Variant> &p_options) {
+	if (p_option.begins_with("animation/")) {
+		if (p_option != "animation/import" &&
+				!bool(p_options["animation/import"])) {
+			return false;
+		}
+	}
+	return true;
+}
+
+void EditorSceneFormatImporterBlend::get_import_options(
+		const String &p_path, List<ResourceImporter::ImportOption> *r_options) {
+	r_options->push_back(ResourceImporter::ImportOption(
+			PropertyInfo(Variant::INT, "blend/nodes/visible",
+					PropertyHint::PROPERTY_HINT_ENUM,
+					"Visible Only,Renderable,All"),
+			BLEND_VISIBLE_ALL));
+	r_options->push_back(ResourceImporter::ImportOption(
+			PropertyInfo(Variant::BOOL, "blend/nodes/punctual_lights"), true));
+	r_options->push_back(ResourceImporter::ImportOption(
+			PropertyInfo(Variant::BOOL, "blend/nodes/cameras"), true));
+	r_options->push_back(ResourceImporter::ImportOption(
+			PropertyInfo(Variant::BOOL, "blend/nodes/custom_properties"), true));
+	r_options->push_back(ResourceImporter::ImportOption(
+			PropertyInfo(Variant::INT, "blend/nodes/modifiers",
```

Comment:
would use blender/ instead of blend/, feels less confusing

Reply:
Will be in the next update.